src/translations/en_US.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   paypalAccountTokenizationFailedError: 'Something went wrong adding the PayPal account. Please try again.',
   paypalFlowFailedError: 'Something went wrong connecting to PayPal. Please try again.',
   paypalTokenizationRequestActiveError: 'PayPal payment authorization is already in progress.',
+  applePayAccountTokenizationFailedError: 'Something went wrong with Apple Pay. Please try again.',
```

Comment:
Didn't see this string used in the code and it isn't an error returned from bt.js